src/nm_database.c

```diff
@@ -125,6 +122,27 @@ void nm_db_select(const char *query, nm_vect_t *v)
     }
 }
 
+void nm_db_select_value(const char *query, nm_str_t *res)
```

Comment:
This function is very similar to `nm_db_select`, can you reuse some of the code somehow? Maybe create a common `nm_db_select(const char *query, int (*cb)(void, int, char**, char **), void *res)`?